python/pycrdt/_awareness.py

```diff
@@ -0,0 +1,165 @@
+from __future__ import annotations
+
+import json
+import time
+from logging import Logger, getLogger
+from typing import Any, Callable
+from uuid import uuid4
+
+from ._doc import Doc
+from ._sync import Decoder, YMessageType, read_message, write_var_uint
+
+DEFAULT_USER = {"username": str(uuid4()), "name": "Jupyter server"}
+
+
+class Awareness:
+    client_id: int
+    log: Logger
```

Comment:
We can get rid of the logger since it's not used.

Reply:
Done